src/libraries/Native/Unix/System.Native/pal_log.c

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include "pal_config.h"
+#include "pal_log.h"
+
+#include <stdio.h>
+
+void SystemNative_Log(uint8_t* buffer, int32_t length)
+{
+    fwrite(buffer, 1, (size_t)length, stdout);
+}
```

Comment:
It might be useful to flush stdout here. Otherwise log messages might not show up on crashes etc.